metadata-ingestion/src/datahub/ingestion/source/sql/sql_config.py

```diff
@@ -90,6 +90,8 @@ class SQLCommonConfig(
     profiling: GEProfilingConfig = GEProfilingConfig()
     # Custom Stateful Ingestion settings
     stateful_ingestion: Optional[StatefulStaleMetadataRemovalConfig] = None
+    # Custom data_dictionary_mode
+    data_dictionary_mode: Optional[str] = None
```

Comment:
This should not be part of SQLCommonConfig. Can you remove this ?

Reply:
@mayurinehate I`ve deleted it from SQLCommonConfig, but mypy returns attr-defined error.